frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -212,23 +212,33 @@ export const eventsTableLogic = kea<eventsTableLogicType<ApiError, EventsTableLo
                 selectors.eventFilter,
                 selectors.orderBy,
                 selectors.properties,
-                selectors.miniumumQueryDate,
+                selectors.minimumQueryDate,
             ],
-            (teamId, eventFilter, orderBy, properties, miniumumQueryDate) =>
+            (teamId, eventFilter, orderBy, properties, minimumQueryDate) =>
                 `/api/projects/${teamId}/events.csv?${toParams({
                     ...(props.fixedFilters || {}),
                     properties: [...properties, ...(props.fixedFilters?.properties || [])],
                     ...(eventFilter ? { event: eventFilter } : {}),
                     orderBy: [orderBy],
-                    after: miniumumQueryDate,
+                    after: minimumQueryDate,
                 })}`,
         ],
-        months: [() => [], () => props.fetchMonths || 12],
-        miniumumQueryDate: [() => [selectors.months], (months) => now().subtract(months, 'months').toISOString()],
+        months: [() => [(_, prop) => prop.fetchMonths], (months) => months || 12],
+        minimumQueryDate: [() => [selectors.months], (months) => now().subtract(months, 'months').toISOString()],
+        getTimestampToQueryAfter: [
+            () => [],
+            () => (timestamp?: string) => {
+                if (!timestamp) {
+                    return now().subtract(5, 'day').toISOString()
+                }
+
+                return dayjs(timestamp).subtract(5, 'day').toISOString()
```

Comment:
Great question! As long as there's some data, every 'load more' sends us back in time 5 days.

As soon as we don't find anymore data, we default to a date range of a year.

What you suggest would work as well, (and might be better for querying less), but I wanted to get the basic system in place first, see it works, and then address any new problems that might come up, if they do (like 'load more' taking too long, because we always default to a year, because no data in past 5 days 🤷 )